src/assets/images/logo.svg

```diff
@@ -0,0 +1,9 @@
+<path d="M323.105 72.0001C321.83 87.3215 309.293 99.643 293.674 100.072C283.155 100.393 273.699 95.2501 268.068 87.2144L321.512 59.5715L332.03 54.1073C325.762 37.8215 309.824 26.4644 291.549 27.0001C268.387 27.643 250.112 47.143 250.749 70.6073C251.387 94.0716 270.83 112.5 293.993 111.857C317.262 111.214 335.537 91.6073 334.899 68.2501C334.899 67.5001 334.793 66.7501 334.793 66.0001L323.105 72.0001ZM262.437 70.2858C261.905 53.3573 275.187 39.3215 291.974 38.7858C301.43 38.4644 309.93 42.643 315.668 49.2858L263.18 76.393C262.755 74.3573 262.543 72.3215 262.437 70.2858Z" fill="#0082CA"/>
+<path d="M672.88 71.5713C671.605 86.8927 659.068 99.2141 643.449 99.6427C632.93 99.9641 623.474 94.8213 617.843 86.7856L671.286 59.1427L681.805 53.6784C675.536 37.3927 659.599 26.0356 641.324 26.5713C618.055 27.2141 599.78 46.8213 600.418 70.1784C601.055 93.6427 620.499 112.071 643.661 111.428C666.93 110.786 685.205 91.1784 684.568 67.8213C684.568 67.0713 684.461 66.3213 684.461 65.5713L672.88 71.5713ZM612.318 69.9641C611.786 53.0356 625.068 38.9999 641.855 38.4641C651.311 38.1427 659.811 42.3213 665.549 48.9641L613.061 76.0713C612.636 74.0356 612.318 71.9999 612.318 69.9641Z" fill="#0082CA"/>
+<path d="M209.21 34.1955L193.247 43.5789C195.209 47.0977 196.368 50.8872 196.546 54.7669C196.546 55.1278 196.546 55.3985 196.546 55.7594C196.546 59.188 195.744 62.6165 194.317 65.7744C185.399 85.1729 151.334 99.6992 110.669 99.6992C105.764 99.6992 100.949 99.5188 96.2221 99.0677C80.4378 97.7143 66.1694 94.1053 54.5764 89.0526L189.769 18.7669C169.615 7.21804 141.346 0 109.955 0C49.2258 0 0 26.8872 0 60C0 74.3459 9.18525 87.5188 24.6129 97.8045C25.9506 98.7068 27.3774 99.609 28.8042 100.421C30.5878 101.504 32.3713 102.496 34.244 103.489C40.3972 106.647 47.1747 109.444 54.4872 111.789C58.0543 112.962 61.7998 113.955 65.6344 114.857C66.3478 115.038 67.0612 115.218 67.7746 115.398C80.7945 118.376 94.9737 120 109.955 120C121.727 120 133.052 119.008 143.664 117.113C164.443 113.504 182.456 106.556 195.833 97.4436C196.101 97.2632 196.368 97.0827 196.546 96.9023C201.451 93.4737 205.732 89.7744 209.21 85.8045C210.547 84.2707 211.707 82.7368 212.866 81.203C215.719 77.1429 217.771 72.812 218.93 68.3008C219.643 65.594 220 62.797 220 60C220 59.3684 220 58.6466 219.911 58.015C219.822 56.9323 219.732 55.7594 219.554 54.6767C219.376 53.6842 219.197 52.6917 218.93 51.6992C217.771 47.188 215.719 42.8571 212.866 38.7068C211.617 37.1729 210.458 35.6391 209.21 34.1955ZM34.244 75.8797C28.0908 69.8346 24.6129 62.9774 24.6129 55.6692C24.6129 55.3083 24.6129 55.0376 24.6129 54.6767C25.683 30.8571 63.6725 11.6391 110.49 11.6391C127.434 11.6391 143.218 14.1654 156.595 18.4962V18.5865L34.244 75.8797Z" fill="#0082CA"/>
+</svg>
```

Comment:
Why you have a lot of paths?

Reply:
It`s default svg logo from official exadel web-site.